### DISCUSSION TITLE
Job Opportunity: Ocean Modeler for mCDR - Oceananigans + OceanBioME

### DISCUSSION THREAD

**Discussion by jperfetto** [CliMA/Oceananigans.jl#3566] [General]
[Ephemeral Carbon](https://ephemeralcarbon.com/) is a new mCDR startup pursuing mineral-based Ocean Alkalinity Enhancement.

We are hiring a PhD-level [Ocean Modeler](https://grnh.se/07850f0b7us) to develop and assimilate data into into biogeochemical models for alkalinity dissolution and sediment transport at the coastal level, and model air/sea re-equilibration at the regional and global levels. We wish to use Oceananigans and OceanBioME for most modeling.

Position is on-site in Santa Clara, CA.

[View more info or apply here](https://grnh.se/07850f0b7us)

**Comment by glwagner** [comment]
Exciting!
